From 20690978d4cf39c1a5e86490ccb2f1e2ae81bcb7 Mon Sep 17 00:00:00 2001
From: Stuart Yoder <stuart.yoder@arm.com>
Date: Mon, 16 Aug 2021 16:56:28 -0500
Subject: [PATCH 30/37] uefi-sct/SctPkg: Secure Boot: improve image loading
 test status codes

add additional status codes to status messages

Signed-off-by: Stuart Yoder <stuart.yoder@arm.com>
---
 .../SecureBoot/BlackBoxTest/ImageLoadingBBTest.c         | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/uefi-sct/SctPkg/TestCase/UEFI/EFI/RuntimeServices/SecureBoot/BlackBoxTest/ImageLoadingBBTest.c b/uefi-sct/SctPkg/TestCase/UEFI/EFI/RuntimeServices/SecureBoot/BlackBoxTest/ImageLoadingBBTest.c
index 7a1ae46b..2bde390b 100644
--- a/uefi-sct/SctPkg/TestCase/UEFI/EFI/RuntimeServices/SecureBoot/BlackBoxTest/ImageLoadingBBTest.c
+++ b/uefi-sct/SctPkg/TestCase/UEFI/EFI/RuntimeServices/SecureBoot/BlackBoxTest/ImageLoadingBBTest.c
@@ -340,7 +340,8 @@ ImageLoadingVariableInit  (
     StandardLib->RecordMessage (
                      StandardLib,
                      EFI_VERBOSE_LEVEL_DEFAULT,
-                     L"Secure Boot ImageLoadingTest: couldn't load dbSigList2.auth\n"
+                     L"Secure Boot ImageLoadingTest: couldn't load dbSigList2.auth. Status=%r.\n",
+                     Status
                      );
     return EFI_NOT_FOUND;
   }
@@ -380,7 +381,8 @@ ImageLoadingVariableInit  (
     StandardLib->RecordMessage (
                      StandardLib,
                      EFI_VERBOSE_LEVEL_DEFAULT,
-                     L"Secure Boot ImageLoadingTest: unable to set db\n"
+                     L"Secure Boot ImageLoadingTest: unable to set db. Status=%r.\n",
+                     Status
                      );
     return EFI_NOT_FOUND;
   }
@@ -415,7 +417,8 @@ ImageLoadingVariableInit  (
     StandardLib->RecordMessage (
                      StandardLib,
                      EFI_VERBOSE_LEVEL_DEFAULT,
-                     L"Secure Boot ImageLoadingTest: unable to load DBXRevokedList1.auth\n"
+                     L"Secure Boot ImageLoadingTest: unable to load DBXRevokedList1.auth. Status=%r.\n",
+                     Status
                      );
     return EFI_NOT_FOUND;
   }
-- 
2.17.1

